src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -557,6 +550,14 @@ public override int Execute(DocumentsOperationContext context)
                     }
 
                     PutAttachments(context, document);
+
+                    //add @HasCounters flag if needed
+                    if (document.TryGetMetadata(out var metadata) && 
+                        metadata.TryGetMember(Raven.Client.Constants.Documents.Metadata.Counters, out _))
+                    {
+                        document.Flags |= DocumentFlags.HasCounters;
```

Comment:
Why is this required? When you'll add the counters, that will get added anyway, no?